lib/src/inline_parser.dart

```diff
@@ -170,7 +170,7 @@ class InlineParser {
             if (d.char == $lbracket) d.isActive = false;
           }
         }
-        _tree[nodeIndex] = linkNode;
+        _tree.replaceRange(nodeIndex, _tree.length, linkNode);
```

Comment:
I don't understand this bit. Does this change anything?

Reply:
now `linkNode` is a list, have to use list operation

Reply:
Oh I see; let's rename the variable, `linkNode` then.